ortools/linear_solver/CMakeLists.txt

```diff
@@ -68,3 +68,18 @@ elseif(UNIX)
 endif()
 
 install(TARGETS solve)
+
+if (BUILD_CXX_EXAMPLES)
+  if (APPLE)
+    set(CMAKE_INSTALL_RPATH
+            "@loader_path/../${CMAKE_INSTALL_LIBDIR};@loader_path")
+  elseif (UNIX)
+    set(CMAKE_INSTALL_RPATH "$ORIGIN/../${CMAKE_INSTALL_LIBDIR}:$ORIGIN:$ORIGIN/../lib:$ORIGIN")
+  endif ()
+
+  add_executable(test_xprs_interface xpress_interface_test.cc)
+  target_compile_features(test_xprs_interface PRIVATE cxx_std_17)
+  target_link_libraries(test_xprs_interface PRIVATE ortools::ortools GTest::gtest_main)
```

Comment:
I can't call `add_cxx_test` because we used `gtest`
Can we add 
`target_link_libraries(test_xprs_interface PRIVATE ortools::ortools GTest::gtest_main)`
to  `add_cxx_test` function in `cmake/cpp.cmake` ?

Reply:
TLDR: yes it's planned but need few fix before...
`add_cxx_test()` is currently only use in examples/tests BUT these files are not googletest. Once release is done i would like to migrate them to googletest (i.e. add gtest_main as target link libraries in the `add_cxx_test()` function.

Reply:
Thanks. I'll propose to keep the `CMakeList.txt` as it is, if it's ok for you.